app/client/views/settings/data/data.js

```diff
@@ -74,29 +78,27 @@ Template.dataSettings.events({
 
     reader.onload = function(fileLoadEvent) {
       Meteor.call("JSONFileImport", reader.result, function(err, data) {
+        if (Array.isArray(data)) {
+          templateInstance.showModal.set(true);
+          templateInstance.importRes.set(data);
+        } else if (!!data.error) {
```

Comment:
What could be some side effects of using `if (data.error)`? It seems simpler, but are there some drawbacks?

Reply:
Alternatively, what about `if (data.error != undefined`)?

Reply:
Better yet, what about `if (error in data)`?